api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 262**:
+
+* [`GET /users/{user_id}/status](/api/get-user-status): Added a new
+  endpoint to get the status of a user with the specified `user_id`.
```

Comment:
This is why you need to **always** look at your documentation changes and post screenshots in your PRs ...

![Screenshot from 2024-05-15 17-11-18](https://github.com/zulip/zulip/assets/63245456/2a1e3628-5227-40cd-957f-6fa9c84e5428)

Reply:
Cool. Thanks for fixing that. So, here again, I think we should make it clear this is something set by the user...

```Added a new endpoint to get a specified user's currently set [status](link-to-help-center).```